src/routers/account.js

```diff
@@ -1,13 +1,25 @@
 const {Router} = require('express');
 const router = Router();
-const checkAuth = require('../middleware/auth')
-const checkRoleAuth = require('../middleware/roleAuth')
-const {getAccounts, postAccount} = require('../controllers/account-controller')
-//const {validationAccount} = require('./validators/validators');
+const checkAuth = require('../middleware/auth');
+const checkRoleAuth = require('../middleware/role-auth');
+const {getAccounts,getAccountsFilters, postAccount, getAccount, putAccount, patchAccount, 
+    validateExistsUsernameEmail, validateExistsAccountUpdate} = require('../controllers/account-controller');
+const {validateExistsCity} = require('../controllers/city-controller');
+const {validationCreationAccount, validationUpdateAccount, validationChangeStatusAccount, validationAccountId, 
+    validationAccountFilters, validationCriterion} = require('../validators/account-validator');
 
 
+router.get('/accounts/:filter/:criterion', checkAuth, checkRoleAuth(["manager"]), validationAccountFilters, validationCriterion, getAccountsFilters);
+
 router.get('/accounts', checkAuth, checkRoleAuth(["manager"]), getAccounts);
-router.post('/accounts', postAccount);
+
+router.get('/accounts/:accountID', checkAuth, checkRoleAuth(["manager", "user"]), validationAccountId, getAccount);
+
+router.post('/accounts', validationCreationAccount, validateExistsUsernameEmail, validateExistsCity, postAccount);
+
+router.put('/accounts', checkAuth, checkRoleAuth(["user"]), validationUpdateAccount, validateExistsCity, validateExistsAccountUpdate, putAccount);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
router.put("/accounts", checkAuth, checkRoleAuth(["user"]), validationUpdateAccount, validateExistsCity, validateExistsAccountUpdate, putAccount);
```